SS14.Launcher/LauncherCommands.cs

```diff
@@ -0,0 +1,144 @@
+using System;
+using System.Buffers;
+using System.Diagnostics;
+using System.IO;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Avalonia.Threading;
+using Serilog;
+using SS14.Launcher.ViewModels;
+using SS14.Launcher.Models.Logins;
+
+namespace SS14.Launcher;
+
+public static class LauncherCommands
```

Comment:
I don't like the fact that these are static.

Reply:
De-staticified